src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -1687,6 +1687,37 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    validator=validate_fs_public_access,
                    help="Specify whether data in the file system may be accessed publicly and the level of access.")
 
+    with self.argument_context('storage fs generate-sas') as c:
+        from .completers import get_storage_acl_name_completion_list
+        t_file_system_permissions = self.get_sdk('_models#FileSystemSasPermissions',
+                                                 resource_type=ResourceType.DATA_STORAGE_FILEDATALAKE)
+        c.register_sas_arguments()
+        c.argument('file_system', options_list=['--name', '-n'], help="File system name.")
+        c.argument('id', options_list='--policy-name',
+                   help='The name of a stored access policy within the container\'s ACL.',
+                   completer=get_storage_acl_name_completion_list(t_container_permissions, 'container_name',
+                                                                  'get_container_acl'))
```

Comment:
Oops, you forgot to change the copied code from container to fs🤣

Reply:
good catch